RELEASE.md

```diff
@@ -209,6 +209,11 @@ This release contains contributions from many people at Google, as well as:
         service workers to preserve existing behavior. The default value is
         `"AUTO"`.
 
+*   Modular File System Migration
+    * S3 and HDFS file system supports have been migrated to modular file systems
+      and is now available in https://github.com/tensorflow/io. The tensorflow-io
+      python package should be installed for S3 and HDFS support with tensorflow.
+
```

Comment:
I'd move this in the breaking changes section

Reply:
Thanks @mihaimaruseac . The PR has been updated.